_config.php

```diff
@@ -35,17 +46,9 @@
             . "|usemap|data*],iframe[src|name|width|height|align|frameborder|marginwidth|marginheight|scrolling],"
             . "object[width|height|data|type],param[name|value],map[class|name|id],area[shape|coords|href|target|alt]"
     ]);
-
-// Re-enable media dialog
-$module = ModuleLoader::inst()->getManifest()->getModule('silverstripe/admin');
-TinyMCEConfig::get('cms')
-    ->enablePlugins([
-        'contextmenu' => null,
-        'image' => null,
-        'sslink' => $module->getResource('client/dist/js/TinyMCE_sslink.js'),
-        'sslinkexternal' => $module->getResource('client/dist/js/TinyMCE_sslink-external.js'),
-        'sslinkemail' => $module->getResource('client/dist/js/TinyMCE_sslink-email.js'),
-    ])
-    ->setOption('contextmenu', 'sslink ssmedia ssembed inserttable | cell row column deletetable');
+// enable ability to insert anchors
+$editorToolbar = $editorConfig->getButtons();
+array_splice($editorToolbar[2], array_search('sslink', $editorToolbar[2]), 0, ['anchor']);
```

Comment:
Is there way you can do this without the magic numbers?

Reply:
ok